v2/src/log_sink_console.c

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/logger.h"
+#include "c_logging/log_context.h"
+
+#include "c_logging/log_sink_console.h"
+
+static void log_sink_console_log(LOG_LEVEL log_level, LOG_CONTEXT_HANDLE log_context, const char* message, const char* file, const char* func, int line)
```

Comment:
>log_sink_console_log [](http://example.com/codeflow?start=12&length=20)

would be awesome if critical/error would be "red" on the console (runs away)

Reply:
This is SuCh A wOnDeRfUl IdEA!!!! Shall take it.

Reply:
Done.